boards/arm/mimxrt1170_evk/Kconfig.defconfig

```diff
@@ -73,4 +76,21 @@ config REGULATOR
 
 endif # DISPLAY
 
+if LVGL
+
+config LV_Z_POINTER_KSCAN
+	default y
+
+config LV_Z_VDB_SIZE
+	default 16
+
+config LV_Z_DPI
+	default 128
+
+choice LV_COLOR_DEPTH
+	default LV_COLOR_DEPTH_16
+endchoice
+
+endif # LVGL
+
```

Comment:
Does this have to be in the board defconfig? Could it be something specific to the screen?

Reply:
There are other display specific configs that could potentially move to a display `shield`. We can take a look as part of a separate PR.